### ISSUE TITLE
Typo on Email Template and Metadata

### ISSUE THREAD

**Issue by drasticdpk** [appirio-tech/connect-app#3860]
### Expected behavior
- In email template it should be technical and crowdsourcing instead of techincal, crowdsorcing respectively.
- In Metadata, it should be quality, Accessibility instead of quallity, Accessibilty on respective page.

### Actual behavior
There is some typo on email Template.
techincal Line no  491 as given file.
crowdsorcing Line no  960 as given file
There is typo in new project creation page.

### Steps to reproduce the problem
- Please refer to attached screenshot.

### Screenshot/screencast

![image](https://user-images.githubusercontent.com/25297859/78763351-c1213d80-79a4-11ea-991f-2f82522ac53c.png)
![image (1)](https://user-images.githubusercontent.com/25297859/78763360-c3839780-79a4-11ea-8ab2-68691edba133.png)
![image (2)](https://user-images.githubusercontent.com/25297859/78763369-c4b4c480-79a4-11ea-952b-909a0ff4946f.png)
[template.html.zip](https://github.com/appirio-tech/connect-app/files/4449374/template.html.zip)

**Comment by drasticdpk** [comment]
Note : Meta data issue only can verify on prod mode.

**Comment by vikasrohit** [comment]
Metadata is fixed on production. Email typo would be reflected on production after 2.9 release.

**Comment by codejamtc** [comment]
![image](https://user-images.githubusercontent.com/42398485/79344712-cb3bc280-7ef5-11ea-8b15-336dea5ecd28.png)
![image](https://user-images.githubusercontent.com/42398485/79344886-02aa6f00-7ef6-11ea-9bfd-7350e3b16a0f.png)

**Need to check the template issue**

**Comment by maxceem** [comment]
@codejamtc it's fixed on DEV now.

@vikasrohit I've noticed that URL for the **Accessibility Testing** Project Template also has the same typo:
![image](https://user-images.githubusercontent.com/146016/79720480-50a1e700-82e9-11ea-8b7d-c85851fc968b.png)


Would you like to get it fixed it on PROD and DEV?

**Comment by vikasrohit** [comment]
Yep, I saw that but I think changing it would require updating links to this form in marketing website which I don't want to do right now. However, I think I can add alias with correct spelling which means it would be accessible with wrong and correct both.

**Comment by codejamtc** [comment]
Metadata is fixed on production. Email typo would be reflected on production after 2.9 release. > Not fixed in Dev or Prod: ❌ 

**Dev:**
![image](https://user-images.githubusercontent.com/42398485/79759713-a88a1f00-82e4-11ea-8b8a-00e7838a7f26.png)

**Prod:**
![image](https://user-images.githubusercontent.com/42398485/79760354-8fce3900-82e5-11ea-88a9-a2085b38e2cd.png)

Dev Spelling issues:  🆗 Fixed

![image](https://user-images.githubusercontent.com/42398485/79757293-84790e80-82e1-11ea-8074-c318929fb396.png)

![image](https://user-images.githubusercontent.com/42398485/79758993-b8edca00-82e3-11ea-836e-0168cd2f7858.png)

URL redirection still pending (prod): ❌ https://connect.topcoder.com/new-project/qa_accessibility_testing - https://monosnap.com/file/9X6qBsTG5VWym2ilbD3IAVS9IKHTb2

**Comment by vikasrohit** [comment]
> URL redirection still pending (prod): ❌ https://connect.topcoder.com/new-project/qa_accessibility_testing - https://monosnap.com/file/9X6qBsTG5VWym2ilbD3IAVS9IKHTb2

It was difficult and not critical to achieve but I have managed to do it in this case. It would load the URL with correct spelling while it would keep loading the incorrect spelled URL as well to support the external links.


> Email typo would be reflected on production after 2.9 release. > Not fixed in Dev or Prod: ❌

Email would be fixed with the 2.9 release on production.

**Comment by vikasrohit** [comment]
@codejamtc typo, in email, is fixed now on production and dev both.

**Comment by codejamtc** [comment]
@vikasrohit I will check both. Thanks

**Comment by codejamtc** [comment]
Fixed in `Dev`: 🆗 
Typo in `Accessibility`: https://monosnap.com/file/npGifQsbzfW8K3KKiXBuvfGBxrPTys
Typo in `crowdsourcing` {Email}: ![image](https://user-images.githubusercontent.com/42398485/80567345-90885e80-89ba-11ea-9478-6ca3a899a6d9.png)

Fixed in `Production`: 🆗 
Typo in `Accessibility`: https://monosnap.com/file/TcL6QzeESvDMM1M40T8vI6VK3vsHmY
Typo in `crowdsourcing` {Email}:

**Comment by drasticdpk** [comment]
verified on prod.
![image](https://user-images.githubusercontent.com/25297859/80568954-9c563e00-8a17-11ea-803b-731a500bbb67.png)
